class.ThemePage.php

```diff
@@ -10,15 +10,6 @@ function __construct($title)
         $root = $_SERVER['DOCUMENT_ROOT'];
         $script_dir = dirname(__FILE__);
         $this->theme_root = substr($script_dir, strlen($root));
-        $this->secureUrl = $this->settings->getGlobalSetting('secure_url', 'https://secure.burningflipside.com');
-        $this->secureMenu = $this->settings->getGlobalSetting('secure_menu', array(
```

Comment:
this appears to be redundant.. 
isn't it rendering the same as is generated by the plugin code in the secure page class?